ansible_rulebook/action/run_job_template.py

```diff
@@ -66,6 +66,9 @@ async def __call__(self):
         )
         await self._job_start_event()
         await self._run()
+        url = self._controller_job_url()
+        if url:
+            print(f"job results url: {url}")
```

Comment:
@jshimkus-rh It might make more sense to 
do a 
logger.info(a_log)
here 
https://github.com/ansible/ansible-rulebook/blob/c6830d0c33069ba1a17814da10f54f32db2e82f1/ansible_rulebook/action/run_job_template.py#L119

That's equivalent to the data being sent up to the server for reporting purposes.